osf/migrations/0136_add_datacite_file_metadata_schema.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-08-16 19:24
+from __future__ import unicode_literals
+
+import json
+import logging
+
+from django.db import migrations
+
+logger = logging.getLogger(__file__)
+
+
+def add_datacite_schema(state, schema):
+    FileMetadataSchema = state.get_model('osf', 'filemetadataschema')
+    with open('osf/metadata/schemas/datacite.json') as f:
+        jsonschema = json.load(f)
+    _, created = FileMetadataSchema.objects.get_or_create(
+        _id='datacite',
```

Comment:
Yep I see your point! I know there was some reasoning behind having the id be a human-readable string. If I remember correctly, it was because we needed to use the `_id` to find the correct schema in the registry. [Here is the decorator](https://github.com/CenterForOpenScience/osf.io/pull/8721/files#diff-898ca0bd378d7ba622a0e1b2c78f2abfR9) and then [in use](https://github.com/CenterForOpenScience/osf.io/pull/8721/files#diff-898ca0bd378d7ba622a0e1b2c78f2abfR34) -- it was essentially so that each `RecordSerializer` knows which schema to use.  

[Here is some of the original discussion](https://github.com/CenterForOpenScience/osf.io/pull/8594#discussion_r209979214) Hooray for notes! This is like a treasure hunt/puzzle made by past-Platform for future-Platform.

Reply:
ah, I see! this makes sense.  it's good as-is for now then. I'm super impressed with the amount of documentation on this feature in general: lots of info in JIRA, lots of info on the subtickets, hooray!